DataFormats/CTPPSReco/interface/CTPPSDiamondLocalTrack.h

```diff
@@ -0,0 +1,94 @@
+/****************************************************************************
+ *
+ * This is a part of CTPPS offline software.
+ * Authors:
+ *   Laurent Forthomme (laurent.forthomme@cern.ch)
+ *   Nicola Minafra nicola.minafra@cern.ch)
+ *
+ ****************************************************************************/
+
+#ifndef DataFormats_CTPPSReco_CTPPSDiamondLocalTrack
+#define DataFormats_CTPPSReco_CTPPSDiamondLocalTrack
+
+#include "DataFormats/Math/interface/Point3D.h"
+#include "DataFormats/CTPPSReco/interface/CTPPSDiamondRecHit.h"
+
+//----------------------------------------------------------------------------------------------------
+
+class CTPPSDiamondLocalTrack
+{
+  public:
+    CTPPSDiamondLocalTrack() :
+      chi_squared_( 0. ), valid_( true ), t_( 0. ), t_sigma_( 0. ), mh_( 0 ) {}
```

Comment:
it would be better to have a constructor with all appropriate arguments to set data members instead of allowing setXXX methods in the interface